.eslintrc

```diff
@@ -1,21 +1,54 @@
 {
+  "globals": {
+    "google": false,
+    "cartodb": false
+  },
   "env": {
     "node": true,
-    "browser": true,
     "es6": true
   },
-  "ecmaFeatures": {
-    "modules": true
+  "parser": "babel-eslint",
+  "parserOptions": {
+    "ecmaVersion": 6,
+    "sourceType": "module",
+    "ecmaFeatures": {
+      "jsx": true
+    }
   },
   "rules": {
-   "indent": [2, 2],
-   "brace-style": [2, "1tbs"],
-   "quotes": [2, "single"],
-   "no-console": 1,
-   "no-use-before-define": [2, "nofunc"],
-   "no-underscore-dangle": 0,
-   "no-constant-condition": 0,
-   "space-before-function-paren": [2, {"anonymous": "always", "named": "never"}],
-   "func-style": [2, "declaration"]
+    "array-bracket-spacing": [2, "never"],
+    "block-spacing": [2, "always"],
+    "brace-style": [2, "1tbs"],
+    "complexity": [1, 10],
```

Comment:
this can, in my experience, make things more complicated when it requires a long but self contained function to be broken up into multiple small functions just to satisfy the rule.